app/src/main/res/values-es/strings.xml

```diff
@@ -103,7 +102,7 @@
     <string name="set_sd_path">Establece aquí la ruta de tu tarjeta SD. Esto es necesario para Android 10 y versiones anteriores.</string>
     <string name="no_sd_card_path_set">¡No se ha establecido la ruta de la tarjeta SD!</string>
     <string name="sd_card_path_set_successfully">¡Ruta de la tarjeta SD establecida con éxito!</string>
-    <string name="clear_sd_card_path">Borrar ruta de la tarjeta SD</string>
```

Comment:
I would recommend to change the resource name/id since now it's more indentificable as "clear". Also the other modified strings